### DISCUSSION TITLE
Encountering an Error while testing the App.

### DISCUSSION THREAD

**Discussion by Suyash878** [abhijeetnishal/URLShortener#117] [Q&A]
## Facing an error while running the App.

I was trying to run the app on my local machine but was encountering this error on my way.

![image](https://github.com/abhijeetnishal/URLShortener/assets/141909257/340aa30c-dc13-48c0-b395-5530678c264f)

**Comment by abhijeetnishal** [comment]
Hey @Suyash878, this issue is due to Redis, there is an issue with the redis check the connection for redis. 
Have you tried shortening the URL after connecting with MongoDB?
